code/datums/station_traits/neutral_traits.dm

```diff
@@ -43,7 +43,8 @@
 	show_in_report = TRUE
 	report_message = "Please be nice to him."
 	blacklist = list(/datum/station_trait/announcement_medbot,
-	/datum/station_trait/announcement_baystation
+	/datum/station_trait/announcement_baystation,
+	/datum/station_trait/announcement_ancestor
```

Comment:
please either identify the changes with a comment as specified in the readme.md file in the austation folder:
OR
override the variables in a modular austation code file
### Comment Formatting
```dm
// austation -- <change reason>
```
for single line changes
and
```dm
// austation start -- <change reason>
<code>
// austation end
```
for multiline